docs/configuration/configuration.md

```diff
@@ -700,6 +700,8 @@ Available meta labels:
 * `__meta_kubernetes_service_port_name`: Name of the service port for the target.
 * `__meta_kubernetes_service_port_number`: Number of the service port for the target.
 * `__meta_kubernetes_service_port_protocol`: Protocol of the service port for the target.
+* `__meta_kubernetes_service_cluster_ip`: The cluster IP address of the service. (Does not apply to services of type ExternalName)
+* `__meta_kubernetes_service_external_name`: The dns-resolvable of the service. (Applies to services of type ExternalName)
```

Comment:
`The DNS name of the service.` instead of `The dns-resolvable of the service.`

Reply:
Done :)